res/qml/Settings/RatioChoice.qml

```diff
@@ -0,0 +1,150 @@
+import "." as Skin
+import Mixxx 1.0 as Mixxx
+import QtQuick 2.12
+import QtQuick.Controls
+import QtQuick.Layouts
+import QtQuick.Shapes
+import Qt5Compat.GraphicalEffects
+import "../Theme"
+import ".." as Skin
+
+Item {
+    id: root
+    required property list<string> options
```

Comment:
```suggestion
    required property list<string> options: []
```
I'm not sure if this is the proper fix, but without it fals very early with:
`warning [Main] file:///D:/mixxx-2.6-worktree/res/qml/Settings/RatioChoice.qml:13:5: Required property options was not initialized`

Reply:
I will look into this. Default on required property don't exist on Qt 6.4, but I'll find a fix for all versions!
Was it working well for you otherwise? I'll share some screen recording of the expected results later today

Reply:
> Was it working well for you otherwise?

No, it failed to start because of missing STEM-Waveform-Renderers later on.

Reply:
It starts with the change above and if I build with STEM enabled, but the sounddsettings page is missing:
![grafik](https://github.com/user-attachments/assets/20aafec1-c85b-4a6a-95e1-6488ebe15579)

Reply:
It seems my patch is not correct:
```
debug [QQmlThread] Processing QEvent::User for object QThread(0x204efebf600, name = "QQmlThread") running in thread: "QQmlThread" took 22 ms
warning [Main] file:///D:/mixxx-2.6-worktree/res/qml/Settings/SoundHardware.qml:176:29: Type RatioChoice unavailable
warning [Main] file:///D:/mixxx-2.6-worktree/res/qml/Settings/RatioChoice.qml:13:5: Required properties with initializer do not make sense.
```

Reply:
I have pushed couple of changes which contains various fix and features to support Windows sound API (I had only tested with ALSA so far). Note that the sound mixer connection edges may appears off at opening, but they should snap to their correct position as soon as you resize the window

Reply:
Not it shows the sound hardware page, but the graphics are not sized properly:
![grafik](https://github.com/user-attachments/assets/4d65d7e1-5f55-4716-a2d7-aadeb9860b07)

Two functional issues:

1. The Sound-API must be selected before Sample Rate and Audio buffer, as the available values depend on th Sound-API. This is not self explaining with the Sound-API on the bottom right.
2. I expected that Main Mix change the graphics if enabled. Is there a need for a Main mixer output, if Main-Mix is disabled?

Reply:
> the graphics are not sized properly

I will look into this. Is that happening constantly or is there steps to reproduce this?

> The Sound-API must be selected before Sample Rate and Audio buffer

I didn't know. I will move that to the  top tab section then

> Is there a need for a Main mixer output, if Main-Mix is disabled?

Ah good shout. I have blindly taken the items from the current setting view, but perhaps it would make sense to remove this option (`Main Mix`) and enable/disable dynamically, depending if it is routed? This would also free a space in the grid and allow `Sound API` to be moved there?

Reply:
It has always some failures, but becomes worse, if I toggle the Simple/Advanced/Legacy switch:
In the beginning only the output connection is shifted:
![grafik](https://github.com/user-attachments/assets/049374c9-6964-49f8-9d86-4f9b6f9ec41b)
After toggling the same mode looks like this;
![grafik](https://github.com/user-attachments/assets/70b1f6a4-1224-4419-84dc-1d244457bb9c)

Reply:
I've pushed some more edge case handling. Let's continue the main thread, as the initial issues here should be fixed.